.gitignore

```diff
@@ -61,6 +61,7 @@ xcuserdata/
 # https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
 
 Pods/
+Podfile.lock
```

Comment:
according to https://guides.cocoapods.org/using/using-cocoapods.html PodFile.lock should not be ignored: "the Podfile and Podfile.lock should always be kept under version control."

Please revert this.